### PR TITLE
show loading bar when checking for entries

### PR DESCRIPTION
**Pull request by agduncan94**
Shows a loading bar when fetching my tools/workflows, instead of showing blank or showing the alert that they have no tools/workflows/

### CODE REVIEW

**Comment by garyluu** [comment]
This type of change can just reuse the refreshMessage variable.  Just set the refreshMessage to something truthy before the usersService call and finalize with something non-truthy

File: src/app/mytools/my-tool/my-tool.component.ts [hunk 1]
```diff
@@ -68,8 +68,10 @@ export class MyToolComponent extends MyEntry implements OnInit {
     this.userService.user$.pipe(takeUntil(this.ngUnsubscribe)).subscribe(user => {
       if (user) {
         this.user = user;
+        this.stateService.setRefreshMessage('Fetching tools');
         this.usersService.userContainers(user.id).pipe(first()).subscribe(tools => {
           this.containerService.setTools(tools);
+          this.stateService.setRefreshMessage(null);
```
Comment (garyluu) [inline src/app/mytools/my-tool/my-tool.component.ts]: put this in finalize or error too

File: src/app/mytools/my-tool/my-tool.component.html [hunk 2]
```diff
@@ -26,7 +26,7 @@
         <mat-sidenav-container autosize class="sidebar-set-height">
           <mat-sidenav #sidenav mode="side" [(opened)]="showSidebar" class="col-sm-3 col-xs-12">
             <span class="visible-xs" *ngIf="refreshMessage">
-              <mat-progress-bar mode="indeterminate"></mat-progress-bar><br />
+              <mat-progress-bar mode="indeterminate" class="m-2"></mat-progress-bar>
```
Comment (garyluu) [inline src/app/mytools/my-tool/my-tool.component.html]: check if this causes the bounce back

Comment (garyluu) [reply src/app/mytools/my-tool/my-tool.component.html]: if it does, copy the progress bar from other components

Comment (agduncan94) [reply src/app/mytools/my-tool/my-tool.component.html]: I meant to put this on the other one, I reverted this back.

**Pull request merged**